base_layer/wallet_ffi/src/error.rs

```diff
@@ -272,6 +272,10 @@ impl From<WalletError> for LibWalletError {
                 code: 427,
                 message: format!("{:?}", w),
             },
+            WalletError::WalletStorageError(WalletStorageError::IncorrectPassword) => Self {
```

Comment:
Shouldn't this account for WalletStorageError::NoPassword as well?

Reply:
I think it's at code 426

Reply:
Nopassword has it's own code above (code 426) - maybe i should have just reused the same code  🤷🏻

Reply:
Ok, was just asking since you did this above on L95 of utilities.rs:
```
Self::IncorrectPassword | Self::NoPassword => 112,
```

Reply:
No, they are semantically different so I'm happy they are separate. Good point @StriderDM - they should be different in there as well